src/stickybits.js

```diff
@@ -245,16 +257,21 @@ class Stickybits {
     toggleClasses ⚖️
     ---
     toggles classes (for older browser support)
-    r = removed class
-    a = added class
+    classes = object of classes, set value to true to add and false to remove
   */
-  toggleClasses (el, r, a) {
-    const e = el
-    const cArray = e.className.split(' ')
-    if (a && cArray.indexOf(a) === -1) cArray.push(a)
-    const rItem = cArray.indexOf(r)
-    if (rItem !== -1) cArray.splice(rItem, 1)
-    e.className = cArray.join(' ')
+  toggleClasses (el, classes) {
```

Comment:
I've reverted the `toggleClasses` changes, however `toggleClasses` is only used in `removeInstance` now. In `applyStyles` it is possible to use `toggleClasses` but it will be bit inconvenient as you have a list of classes to add/remove now.